frontend/src/components/dashboard/incidentsHomePage/IncidentsHomePage.js

```diff
@@ -110,20 +111,31 @@ export class IncidentsHomePage extends Component {
     this.state = {
       incidentsLoaded: false,
       params: {
+        page: Strings.DEFAULT_PAGE,
+        per_page: Strings.DEFAULT_PER_PAGE,
         region: formatParamStringToArray(this.params.region),
         major: this.params.major,
         search: this.params.search,
         year: this.params.year,
         incident_status: this.params.incident_status,
         codes: this.params.codes,
         determination: this.params.determination,
+        ...this.params,
       },
```

Comment:
what is left in this spread? everything should be defined in the state

Reply:
removed